From 1f840fc96b1a1ff86b0d2985b8c9a3874bd4567f Mon Sep 17 00:00:00 2001
From: Laurent Bigonville <bigon@debian.org>
Date: Thu, 26 Dec 2013 14:45:28 +0100
Subject: [PATCH] Switch to "avahi-daemon" for the LSB initscript dependency
 (Closes: #731608)

Also add a Breaks against previous version of avahi-daemon package
---
 debian/control               | 2 +-
 debian/cups-daemon.cups.init | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/debian/control b/debian/control
index f13e2e0..ab25dc4 100644
--- a/debian/control
+++ b/debian/control
@@ -204,7 +204,7 @@ Suggests: cups, cups-bsd,
  udev,
  smbclient
 Replaces: cups (<< 1.6.1-1~)
-Breaks: cups (<< 1.6.1-1~)
+Breaks: cups (<< 1.6.1-1~), avahi-daemon (<< 0.6.31-3~)
 Multi-Arch: foreign
 Description: Common UNIX Printing System(tm) - daemon
  The Common UNIX Printing System (or CUPS(tm)) is a printing system and 
diff --git a/debian/cups-daemon.cups.init b/debian/cups-daemon.cups.init
index 06a6d8f..f645cc2 100644
--- a/debian/cups-daemon.cups.init
+++ b/debian/cups-daemon.cups.init
@@ -3,7 +3,7 @@
 # Provides:          cups
 # Required-Start:    $syslog $remote_fs
 # Required-Stop:     $syslog $remote_fs
-# Should-Start:      $network avahi slapd nslcd
+# Should-Start:      $network avahi-daemon slapd nslcd
 # Should-Stop:       $network
 # X-Start-Before:    samba
 # X-Stop-After:      samba
-- 
1.8.5.2

